tutorials/contour/index.pug

```diff
@@ -0,0 +1,81 @@
+extends ../common/index.pug
+
+block mainTutorial
+  :markdown-it
+    # Tutorial - Contour map
```

Comment:
It looks like I've mostly capitalized things in other tutorial titles, so can this be `Contour Map`.  Also change in the tutorial.json file and do similarly for the Simple Point tutorial as well.